src/components/bs5/header/header.hbs

```diff
@@ -237,7 +237,7 @@
                 {{/if}}
                 <button aria-controls="main-nav" class="qld__header__toggle-main-nav qld__main-nav__toggle--open"
                     data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
-                    aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Open menu">
```

Comment:
Here as well. Need to be 'Toggle menu'.

Reply:
Changes is reflecting the accessibility test: https://ssq-qol.atlassian.net/browse/QOLOE-412